src/test/randomizeArray.test.js

```diff
@@ -0,0 +1,7 @@
+import randomizeArray from "../helperFunctions/randomizeArray";
+
+test("randomizeArray creates an array of random numbers", () => {
```

Comment:
be more specific of what the array may look like in the description:

"creates an array of random numbers from `0 to n` when `n` is passed as an argument"